echopype/tests/convert/test_convert_azfp.py

```diff
@@ -161,3 +161,17 @@ def test_convert_azfp_01a_different_ranges(azfp_path):
 
     # check convention-required variables in the Platform group
     check_platform_required_vars(echodata)
+
+
+def test_convert_azfp_01a_notemperature(azfp_path):
+    """Test converting file with no temperature data."""
+    azfp_01a_path = str(azfp_path.joinpath('rutgers_glider_notemperature', '22052500.01A'))
+    azfp_xml_path = str(azfp_path.joinpath('rutgers_glider_notemperature', '22052501.XML'))
```

Comment:
question: wouldn't the code work without the use of `str`?

Reply:
I started the new test code by copying and pasting one of the existing tests in that module. Yes, `str` is unnecessary, but its use is also present in that context in every test in the module. I'd rather deal with that in the context of a broader clean up.

But feel free to push a commit that cleans this up :wink:

Reply:
haha, I am going to push back and say: how about clean up just this test file? It's unclear to me when a broader clean up will happen, so might as well do small bits first.

Reply:
Addressed in a new commit.